### PR TITLE
[wip] init google service

### PR DESCRIPTION
**Pull request by mauriyouth**

### CODE REVIEW

File: front/components/app/ModelPicker.jsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ export default function ModelPicker({ user, model, readOnly, onModelUpdate }) {
                 )}
               >
                 <div className="py-1">
-                  {providers.map((p) => {
+                  {modelProviders.map((p) => {
```
Comment (spolu) [inline front/components/app/ModelPicker.jsx:143]: nice catch

File: front/pages/[user]/providers/index.jsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { classNames } from "../../../lib/utils";
 import { Button } from "../../../components/Button";
 import OpenAISetup from "../../../components/providers/OpenAISetup";
 import CohereSetup from "../../../components/providers/CohereSetup";
+import GoogleSetup from "../../../components/providers/GoogleSetup";
```
Comment (spolu) [inline front/pages/[user]/providers/index.jsx:10]: This file is missing I think?

**Review by spolu — COMMENTED** [review]
Looking good!

I think the overarching question I haven't answered is, Google Custom Search or SerpApi.

SerpApi is more vestatile for sure and you get cards answers which is kind of nice for some demo use-cases.

Any feeling on this now that you've digged a little?

File: front/pages/[user]/providers/index.jsx (hunk 2 above)
Comment (mauriyouth) [reply front/pages/[user]/providers/index.jsx:10]: yep, added missing files

**Pull request closed without merging**